services/director-v2/src/simcore_service_director_v2/api/dependencies/dynamic_services.py

```diff
@@ -21,14 +21,10 @@ async def get_service_base_url(
 ) -> URL:
 
     # get the service details
-    service_details: RunningServiceDetails = (
+    service_details: RunningDynamicServiceDetails = (
```

Comment:
did you search for all occurences?
still is visible in openapi.json. maybe you should upgrade that one too with ```make openapi-specs```